lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +18,22 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+
+    if any(symb.isalpha() for symb in text) is False:
+        return []
+    new_text = []
+    word_for_new_text = []
+
+    for every_symbol in text.lower():
+        if every_symbol.isalpha():
```

Comment:
use continue

Reply:
where?.. Please, explain

Reply:
in the 'for construction' there are two brunches that cannot be replaced with 'continue', none of them

Reply:
now seems to be done